src/client/debug/log/LogEvent.svelte

```diff
@@ -1,8 +1,8 @@
 <script>
   export let logIndex;
-  export let onLogClick;
-  export let onMouseEnter;
-  export let onMouseLeave;
+  //export let onLogClick;
+  //export let onMouseEnter;
+  //export let onMouseLeave;
```

Comment:
Why is this changing?

Reply:
They are unused and thus the svelte compiler warns on build.

Reply:
I see.  Let's delete them, then.

Reply:
Removed them.